eclair-core/src/main/resources/reference.conf

```diff
@@ -38,6 +38,7 @@ eclair {
   // see https://github.com/lightningnetwork/lightning-rfc/blob/master/09-features.md
   features {
     option_data_loss_protect = optional
+    option_upfront_shutdown_script = optional
```

Comment:
I think this should be disabled by default.
We should also add an option to provide the final script in the `open` API.
Finally, we should add a big comment here explaining the behavior:
- If users activate this feature, they will need to either provide a script when opening the channel, or accept that the funds will go back to the bitcoin core wallet that's loaded at the moment of opening (via our default final script)

Reply:
Activating this feature was definitely not a good idea, I'll leave it disabled by default. 
I'm not sure yet we should provide a way for users to specify a custom shutdown script: we rely on bitcoin core to manage our node's wallet, and for example we use wallet points when static_remote_key is enabled.